vision/google/cloud/vision/color.py

```diff
@@ -93,6 +111,22 @@ def from_api_repr(cls, response):
 
         return cls(red, green, blue, alpha)
 
+    @classmethod
+    def from_pb(cls, color):
+        """Factory: construct a ``Color`` from a protobuf response.
+
+        :type color: :class:`~google.type.color_pb2`
```

Comment:
This is a module, not a class (not that it will show up via Intersphinx either way)